requirements.txt

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: MPL-2.0
 
 geopy~=2.1.0
-openstef~=3.2.0
+openstef~=3.2.1
```

Comment:
The github action 'build' says that this version does not exist:
![image](https://user-images.githubusercontent.com/18208480/165691193-56d26fca-17a4-414b-8fef-74e5327fb8be.png)

Reply:
Yes will have to wait for the release of openstef

Reply:
That release is nearly ready :)